src/operators/frei0r_op.ml

```diff
@@ -146,7 +147,7 @@ class frei0r_mixer ~kind ~name bgra instance params (source : source) source2 =
               Frei0r.update2 instance t src src' dst;
               if bgra then Image.RGBA32.swap_rb img;
               let img = Image.YUV420.of_RGBA32 img in
-              Video.set rgb i img;
+              Video.Canvas.put rgb i img;
```

Comment:
What's the order of operations here? I'd expect the data to be an array of canvas but this API seems a bit confusing to me?

Reply:
Essentially
```
Video.Canvas.put rgb i img
```
is the same as
```
rgb.(i) <- img
```
excepting that we cast the `img` (which is a proper image) into a canvas.